tests/playTests.sh

```diff
@@ -310,6 +310,17 @@ test -f precompressedFilterTestDir/input.6.zst.zst
 println "Test completed"
 
 
+
+println "\n===>  warning prompt does not swallow characters"
+println "y" > tmpPrompt
+println "hello world" >> tmpPrompt
+zstd tmpPrompt
+zstd < tmpPrompt -o tmpPrompt.zst
```

Comment:
My understanding of this scenario : 
- `zstd tmpPrompt` creates `tmpPrompt.zst`.
- `zstd < tmpPrompt -o tmpPrompt.zst` tries to basically overwrite `tmpPrompt.zst`
- since there is no `-f`, it can't overwrite without confirmation
- since input is `stdin` it can't ask for overwrite confirmation
- since it can't ask for overwrite confirmation, it should automatically fail

This is not what I see : it seems this test is intended to succeed, hence it would overwrite `tmpPrompt.zst`,
which, I believe, is not correct for this use case.

Thoughts ?